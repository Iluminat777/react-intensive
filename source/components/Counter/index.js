//Core
import React, { Component } from 'react';

// Instruments
import Style from './style.m.css';


// Components

const Counter = ({ counter }) =>
    <section className = { Style.counter }> {`Posts count: ${counter}`} </section>;

export default Counter;