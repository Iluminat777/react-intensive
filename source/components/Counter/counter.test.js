//Core
import React from 'react';
import Counter from './index';
import dom from 'react-test-renderer';


const renderTree = dom.create(<Counter counter = { 9 } />).toJSON();

test('Counter component should correspond to their snapshot', () => {
    expect(renderTree).toMatchSnapshot();
});