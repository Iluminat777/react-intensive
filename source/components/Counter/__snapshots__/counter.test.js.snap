// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component should correspond to their snapshot 1`] = `
<section
  className="counter"
>
   
  Posts count: 9
   
</section>
`;
